server/util/pbwireutil/pbwireutil_test.go

```diff
@@ -0,0 +1,39 @@
+package pbwireutil_test
+
+import (
+	"testing"
+
+	"github.com/buildbuddy-io/buildbuddy/server/util/pbwireutil"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/protobuf/proto"
+
+	repb "github.com/buildbuddy-io/buildbuddy/proto/remote_execution"
+	tspb "github.com/golang/protobuf/ptypes/timestamp"
+)
+
+func TestWireUtils(t *testing.T) {
+	msg := &repb.ExecutedActionMetadata{
+		Worker:          "foo",
+		QueuedTimestamp: &tspb.Timestamp{Seconds: 7},
+	}
+	b, err := proto.Marshal(msg)
+	require.NoError(t, err)
+
+	const (
+		mdWorkerFieldNumber          = 1
+		mdQueuedTimestampFieldNumber = 2
+
+		tsSecondsFieldNumber = 1
+	)
+	// Get string
+	{
+		worker, _ := pbwireutil.ConsumeFirstString(b, mdWorkerFieldNumber)
+		require.Equal(t, worker, "foo")
```

Comment:
These days my preference is not to use `assert`, because I feel like it adds little benefit - test failures need to be fixed one-by-one anyway, and it's an extra decision to have to make when writing code (do I use assert or require here?), and it's another thing that reviewers have to look for (for each assert/require, they have to think about whether it's "correct"). Overall I feel like just using require everywhere is much simpler.

Reply:
![image](https://github.com/buildbuddy-io/buildbuddy/assets/141737/6a0cb772-cf2b-4d01-91a8-a2e4a17aed4c)

What if we just stopped using testify altogether and used t.Fatalf?